code/modules/mob/living/silicon/robot/examine.dm

```diff
@@ -6,6 +6,10 @@
 	var/obj/act_module = get_active_held_item()
 	if(act_module)
 		msg += "It is holding [icon2html(act_module, user)] \a [act_module].\n"
+	for(var/V in status_effects)
+		var/datum/status_effect/E = V
+		if(E.examine_text)
+			msg += "[replacetext(E.examine_text, "SUBJECTPRONOUN", "It")]\n"

```

Comment:
Robots actually respect pronouns correctly. Also, this text should be in base examine *technically* but examine code sucks. Make it a proc.

Reply:
The replacement of SUBJECTPRONOUN or other text with the correct pronoun for the mob?

Reply:
No, the status effects showing up in examine.

Reply:
Ohh, good point.